drain-service/drain_training_inferencing.py

```diff
@@ -326,6 +328,20 @@ def weighted_avg_and_std(values, weights):
 
                 training_start_ts_ns = time.time_ns()
 
+                drain_status_doc = {
+                    "num_log_clusters": num_drain_templates,
+                    "update_type": "training_signal",
+                    "timestamp": int(training_end_ts_ns / 1000000),
+                }
+                try:
+                    await es.index(
+                        index="opni-drain-model-status", body=drain_status_doc
```

Comment:
Is it worth versioning the index?  That way if we change the index template down the track we can migrate to a new index.